### ISSUE TITLE
Issue in running model

### ISSUE THREAD

**Issue by RavneetDTU** [abhirooptalasila/AutoSub#68]
Command I run: python3 autosub/main.py --file video.mp4

[INFO] ARGS: Namespace(dry_run=False, engine='stt', file='video.mp4', format='srt', model=None, scorer=None, split_duration=5)
[INFO] Model: /media/ravneet/SSD2/TMN_Tasks/AutoSub/model.tflite
[INFO] Scorer: /media/ravneet/SSD2/TMN_Tasks/AutoSub/deepspeech-0.9.3-models.scorer
[INFO] Input file: video.mp4
[INFO] Extracted audio to audio/video.wav
[INFO] Splitting on silent parts in audio file
[INFO] Running inference...
TensorFlow: v2.3.0-6-g23ad988fcde
 Coqui STT: v0.10.0-alpha.10-0-g9b517632
Warning: reading entire model file into memory. Transform model file into an mmapped graph to reduce heap usage.
2022-08-15 23:53:27.304189: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Data loss: Can't parse /media/ravneet/SSD2/TMN_Tasks/AutoSub/model.tflite as binary proto
[ERROR] Invalid model file

**Comment by abhirooptalasila** [comment]
Hi
I need more info about your setup. Where did you download the model files from? By default Autosub uses Coqui for STT. Did you download those models or DeepSpeech'?
